airbyte-config/init/build.gradle

```diff
@@ -1,9 +1,29 @@
+import com.bmuschko.gradle.docker.tasks.image.DockerBuildImage
+
 plugins {
     id 'java'
+    id 'com.bmuschko.docker-remote-api'
 }
 
 dependencies {
     implementation 'commons-cli:commons-cli:1.4'
 
     implementation project(':airbyte-config:models')
 }
+
+task copyScripts(type: Copy) {
+    from('scripts')
+    into 'docker/bin/scripts'
+}
+
+def buildTag = System.getenv('VERSION') ?: 'dev'
+// Use task types
+task buildMyAppImage(type: DockerBuildImage) {
+    dependsOn copyScripts
+    dependsOn compileTestJava
+    dependsOn jar
+    dependsOn processTestResources
+    inputDir = file('docker')
```

Comment:
While this isn't strictly necessary since the default is the `docker` dir, I do like keeping the input here since it isn't a particularly obvious default.